spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/tracing/otlp/OtlpProperties.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright 2012-2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.boot.actuate.autoconfigure.tracing.otlp;
+
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * Configuration properties for {@link OtlpAutoConfiguration}.
+ *
+ * @author Jonatan Ivanov
+ * @since 3.1.0
+ */
+@ConfigurationProperties("management.otlp.tracing")
+public class OtlpProperties {
+
+	/**
+	 * Whether auto-configuration for OTLP exporter is enabled.
+	 */
+	private boolean enabled = true;
+
+	/**
+	 * HTTP configuration.
+	 */
+	private final Http http = new Http();
+
+	public boolean isEnabled() {
+		return this.enabled;
+	}
+
+	public void setEnabled(boolean enabled) {
+		this.enabled = enabled;
+	}
+
+	public Http getHttp() {
+		return this.http;
+	}
+
+	public static class Http {
```

Comment:
I'm not sure about this `Http` grouping, particularly if it turns out that the `enabled` property isn't needed. The main reason I'm not sure about it is that it's not consistent with what we've done for Zipkin where we have `management.tracing.zipkin.connect-timeout`, `management.tracing.zipkin.endpoint`, `management.tracing.zipkin.read-timeout`.

Reply:
Both `io.opentelemetry.exporter.otlp.http.trace.OtlpHttpSpanExporterBuilder` and `io.opentelemetry.exporter.otlp.trace.OtlpGrpcSpanExporterBuilder` have basically the same properties, so that also goes in favor of not having the grouping and reusing the same configuration properties for both HTTP and gRPC if the enum approach ends up being preferred.

Reply:
👍🏼